consensus/state.go

```diff
@@ -1583,7 +1583,7 @@ func (cs *State) finalizeCommit(height int64) {
 		panic(fmt.Errorf("+2/3 committed an invalid block: %w", err))
 	}
 
-	logger.Info(
+	logger.Debug(
```

Comment:
I personally think this should be info. My take on the things that we want to know at an info basis is:

- When we sign a proposal
- When we vote
- When we finalize a block/commit

Reply:
```
8:11AM INF finalizing commit of block hash=E27ADB844E9189F2EF2B37F26102BF1574849C62F64ABE74D1B9FFE8B4BD4167 height=5355584 module=consensus num_txs=1 root=4CD95268DB9C8CB1CCFEDBDCEDB2F61D801625426C775A1650ECEADE8D9A00EB
8:11AM INF executed block height=5355584 module=state num_invalid_txs=0 num_valid_txs=1
```

Reason for removal is these two logs say the same thing essentially, IMO. One gets fired 20 seconds before the other.

Reply:
Sure but they do represent different things right: first I want to know that we have agreed on something, then I want to know that I successfully executed that block

Reply:
I agree if you are a developer of the software. 

Note this wasn't logged till recently. 

As a node operator I think you would want to receive clean logs, this log doesn't add much value and will most likely be discarded by log capture systems.

Reply:
Yeah I think what defines a clean log is something that we all need to reach consensus on. 

I'm not a node operator so I'm not sure what exactly they use logs for but I believe most of the node operators wouldn't actually be looking at logs but instead, as you say, send them to some automated log processing thing. 

So sure, what the node operators want to know is that something is running smoothly. But if the shit hits the fan it's us as the developer that's going to be reading these logs. If the logs they send contain nothing more but an error and everything before that is just "we executed block x", then I'm not sure that we will always have enough context to know what went wrong.

Reply:
> just "we executed block x", then I'm not sure that we will always have enough context to know what went wrong.

This is how it was in tendermint till recently, I believe. I don't think this affects our debugging as we would ask for a consensus dump because the logs can only tell so much no matter what. If we dont finalize the block then we wouldn't see ` executed block x`. I'm fine to add it back, just didn't see any value in it.

Reply:
I think a consensus dump is only helpful for a subset of possible errors. But you're right, the default before wasn't particularly helpful if someone was to give us those logs. 

> just didn't see any value in it.

The value I saw was that it could help me point to whether the problem occurred in consensus itself or whether it occurred in the block executor. Maybe this isn't too valuable. I also didn't want to start this entire "what is the meaning of logs" philosophical debate. So I'm easy but basically I'm asking myself what is the minimum level of information we need to help us debug the largest amount of possible errors

Reply:
I personally think this should remain INFO.

Reply:
Then would propose move executed block to DEBUG. I fail to see the value in both of the logs.

Reply:
Sure 👍